orchagent/main.cpp

```diff
@@ -227,7 +231,20 @@ int main(int argc, char **argv)
         case 'z':
             sai_deserialize_redis_communication_mode(optarg, gRedisCommunicationMode);
             break;
-
+        case 'f':
+            swss_rec_filename = optarg;
+            if (swss_rec_filename.empty())
+            {
+                swss_rec_filename=  "swss.rec";
+            }
+            break;
```

Comment:
How about changing the value of `swss_rec_filename` only when `optarg` is not empty? Since the default is "swss.rec", it seems unnecessary to change the value to empty and then back to "swss.rec".

Reply:
Fixed in latest commit